File: 프로그래머스/2/12945. 피보나치 수/피보나치 수.py
def solution(n):
    dp = [0] * (n + 1)
    
    for i in range(1, n + 1):
        if i == 1 : dp[i] = 1
        else: dp[i] += dp[i-1] + dp[i-2]
    
    return dp[n] % 1234567

File: 프로그래머스/2/12945. 피보나치 수/피보나치 수.js
function solution(n) {
    let f = Array(n+1).fill(0)
    f[1] = 1
    
    for(let i = 2; i <= n; i++){
        f[i] = (f[i-2] + f[i-1]) % 1234567
    }
    
    return f[n] ;
}